libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -754,6 +754,53 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 		}
 		break;
 	}
+	case FunctionType::Kind::Creation:
+	{
+		solAssert(!functionType->gasSet(), "Gas limit set for contract creation.");
+		solAssert(
+			functionType->returnParameterTypes().size() == 1,
+			"Constructor should return only one type"
+		);
+
+		TypePointers argumentTypes;
+		string constructorParams;
+		for (auto const& arg: arguments)
```

Comment:
The definition of `arguments` is very far away. Would it hurt to use an explicit type here? :)

Reply:
Ah, sometimes I am too lazy and just hit auto and then forget to change it to real type, need to be more disciplined :)